scripts/ci/get_image_tags_from_branch.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -xe
+GIT_BRANCH=$1
+IMAGE_VERSION=$2
+BUILD_NUMBER=$3
+COMMIT_HASH=$4
+branch_tag=`echo $GIT_BRANCH| sed 's|/|.|g'`  #not sure if docker accept / in the version
+specific_tag="${IMAGE_VERSION}_b${BUILD_NUMBER}_${COMMIT_HASH}${branch_tag}"
```

Comment:
I feel like we are [doing double work](https://github.com/IBM/ibm-block-csi-operator/pull/189#discussion_r693491353) here

Reply:
I aligned it, but the only better solution is to combine the two repositories into one

Reply:
or just not to open more than one PR with common content in the same time

Reply:
I won't delay PR because some files (the other PR is not taking two days:shrug:)

Reply:
maybe if we focus on 1 PR at a time, each PR would end quicker